examples/product-documents/ecs/usage-examples/python/demo/deploy_svn_by_using_svnserve_on_ECS.py

```diff
@@ -0,0 +1,167 @@
+import ros_cdk_core as core
+import ros_cdk_ecs as ecs
+
+
+class ECS_SVN_svnserve_CentOS_7(core.Stack):
+
+    def __init__(self, scope: core.Construct, construct_id: str, **kwargs) -> None:
+        super().__init__(scope, construct_id, **kwargs)
+
+        """
+        Note：
+        The following code only cover the deployment and configuration part of SVN in the documentation.        
+        
+        The ECS created in the script has the following configuration:
+        Instance type: ecs.c6.large(default value), you may choose other instance types in the template
+        CentOS 7.2 64-bit public image
+        Network type：VPC
+        Security group：Inbound rules are added to the security groups of the instance to allow traffic on port 22, 80, 443, 3690
+        Public IP: assigned
+        
+        SVN: 1.7.14
+
+        """
+
+        core.RosInfo(self, core.RosInfo.description, "Script created by Yunjie Peng")
```

Comment:
This should be a description of the CDK script, i.e. what the stack is used for.

Reply:
Updated the description.